doc/Makefile

```diff
@@ -47,11 +47,12 @@ api:
 
 release_notes:
 	@echo "Copying release notes"
-	@cat `$(PYTHON) -c \
-		"from pathlib import Path; from skimage import io; \
-		 l = list(str(e) for e in Path('release').glob('release_*.*.*')); \
-		 l.sort(key=io.collection.alphanumeric_key); print(l[-1])"` \
-		| tail -n +4 > release/_release_notes_for_docs.rst
+	@tail -n +4 release/release_$$(\
```

Comment:
Could you please explain why you made this change? I understand that it's cleaner to use shell commands instead of calling python here but on the other hand folks here are more familiar with Python so the "ugly" version with Python might be easier to maintain. Not a strong feeling here, just trying to decide what is best.

Reply:
Yeah, I made this change because the shell expansion of `$(PYTHON)` is getting mangled and was breaking the build. The code is being executed on Windows by a variation of Bash (it seems to be different per Python version), but since Python is coming from Windows it needs to be specified as a full path to the version of Python so it's not picked up by bash. The issue is coming from either Make or Bash, but the Python code is also "not very straight forward" as to what it is doing (maybe this should be not an inline command) and I figured it would not be a big deal to change it to a roughly equally "hard to read" shell command which would keep everything using whatever version of Bash this was running on.